src/transform/sanitize.ts

```diff
@@ -0,0 +1,486 @@
+    'dy',
+    'diffuseconstant',
+    'direction',
+    'display',
+    'divisor',
+    'dur',
+    'edgemode',
+    'elevation',
+    'end',
+    'fill',
+    'fill-opacity',
+    'fill-rule',
+    'filter',
+    'filterunits',
+    'flood-color',
+    'flood-opacity',
+    'font-family',
+    'font-size',
+    'font-size-adjust',
+    'font-stretch',
+    'font-style',
+    'font-variant',
+    'font-weight',
+    'fx',
+    'fy',
+    'g1',
+    'g2',
+    'glyph-name',
+    'glyphref',
+    'gradientunits',
+    'gradienttransform',
+    'height',
+    'href',
+    'id',
+    'image-rendering',
+    'in',
+    'in2',
+    'k',
+    'k1',
+    'k2',
+    'k3',
+    'k4',
+    'kerning',
+    'keypoints',
+    'keysplines',
+    'keytimes',
+    'lang',
+    'lengthadjust',
+    'letter-spacing',
+    'kernelmatrix',
+    'kernelunitlength',
+    'lighting-color',
+    'local',
+    'marker-end',
+    'marker-mid',
+    'marker-start',
+    'markerheight',
+    'markerunits',
+    'markerwidth',
+    'maskcontentunits',
+    'maskunits',
+    'max',
+    'mask',
+    'media',
+    'method',
+    'mode',
+    'min',
+    'name',
+    'numoctaves',
+    'offset',
+    'operator',
+    'opacity',
+    'order',
+    'orient',
+    'orientation',
+    'origin',
+    'overflow',
+    'paint-order',
+    'path',
+    'pathlength',
+    'patterncontentunits',
+    'patterntransform',
+    'patternunits',
+    'points',
+    'preservealpha',
+    'preserveaspectratio',
+    'primitiveunits',
+    'r',
+    'rx',
+    'ry',
+    'radius',
+    'refx',
+    'refy',
+    'repeatcount',
+    'repeatdur',
+    'restart',
+    'result',
+    'rotate',
+    'scale',
+    'seed',
+    'shape-rendering',
+    'specularconstant',
+    'specularexponent',
+    'spreadmethod',
+    'startoffset',
+    'stddeviation',
+    'stitchtiles',
+    'stop-color',
+    'stop-opacity',
+    'stroke-dasharray',
+    'stroke-dashoffset',
+    'stroke-linecap',
+    'stroke-linejoin',
+    'stroke-miterlimit',
+    'stroke-opacity',
+    'stroke',
+    'stroke-width',
+    'style',
+    'surfacescale',
+    'systemlanguage',
+    'tabindex',
+    'targetx',
+    'targety',
+    'transform',
+    'text-anchor',
+    'text-decoration',
+    'text-rendering',
+    'textlength',
+    'type',
+    'u1',
+    'u2',
+    'unicode',
+    'values',
+    'viewbox',
+    'visibility',
+    'version',
+    'vert-adv-y',
+    'vert-origin-x',
+    'vert-origin-y',
+    'width',
+    'word-spacing',
+    'wrap',
+    'writing-mode',
+    'xchannelselector',
+    'ychannelselector',
+    'x',
+    'x1',
+    'x2',
+    'xmlns',
+    'y',
+    'y1',
+    'y2',
+    'z',
+    'zoomandpan',
+    'from',
+    'to',
+];
+
+const allowedTags = Array.from(
+    new Set([...htmlTags, ...svgTags, ...sanitizeHtml.defaults.allowedTags]),
+);
+
+const allowedAttributes = Array.from(new Set([...htmlAttrs, ...svgAttrs]));
+const allowedSchemes = ['conisio', 'jamfselfservice', 'softwarecenter'];
```

Comment:
I think these schemes are not needed all by default

Reply:
I remove them